src/ORM/Table.php

```diff
@@ -755,7 +754,7 @@ public function belongsTo($associated, array $options = [])
         $options += ['sourceTable' => $this];
         $association = new BelongsTo($associated, $options);
 
-        return $this->_associations->add($association->name(), $association);
+        return $this->_associations->add($association->getName(), $association);
```

Comment:
Please avoid making unrelated changes in a PR. It adds noise and shifts focus from task at hand.